basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
```

Comment:
Maybe something for another iteration, but how about taking in an app ID and automatically finding the path to the shard?